packages/subproviders/test/utils/ganache_subprovider.ts

```diff
@@ -15,4 +15,4 @@ export const ganacheSubprovider = new GanacheSubprovider({
     port: configs.port,
     networkId: configs.networkId,
     mnemonic: configs.mnemonic,
-});
```

Comment:
why `as any`?

Reply:
Typescript isn't determining that these are the same types since Subprovider is redeclared in dev-utils. I'll add a comment about this.